### PR TITLE
Generate one Python package per pipeline.  

### PR DESCRIPTION
**Pull request by jsilberm**
Changes to Makefile for generating one package per pipeline.
Allow TargetDirectory for generated code (default is current directory).
Changes to `getswagger.py` to store swagger files on a per pipeline basis.
Added `mkpypigenconfig.sh` to dynamically generate the code generation scripts on a per pipeline basis.
Pointing PyPi metadata to `https://github.com/pensando/pypi` for the repo.
`default` Makefile rule changed to build for all 3 pipelines.   Can be overridden, of course (Ex:  `make pypi_cloud`, `make genconfig_dss`)

### CODE REVIEW

File: pyclient/mkpypigenconfig.sh [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    "modelPackage": "model",
+    "apiPackage": "api",
+    "allgroups" : `mkallgroups $1`
+ }
+EOF
+
+}
+
+function mkgensh() {
+dirname=${destdir}/src_${1}/`cat genconfig_${1}.json | jq .libName |sed -e 's^"^^g'`
```
Comment (RishabhPatil) [inline pyclient/mkpypigenconfig.sh:41]: The container doesnt have 'jq' installed as of now, that's the only issue I'm running into.

**Review by RishabhPatil — CHANGES_REQUESTED** [review]
Can you add 'jq' to the list of installed packages in `Dockerfile`

File: pyclient/mkpypigenconfig.sh [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+done
+
+echo "[$allgroups]"
+}
+
+function mkgenconfig() {
+[ ! -d $src_dirname ] && echo "Creating $src_dirname ..."; mkdir $src_dirname
+
+echo Creating genconfig_$1.json ...
+cat <<EOF > genconfig_$1.json
```
Comment (RishabhPatil) [inline pyclient/mkpypigenconfig.sh:28]: I think these new generated files should be added to the .gitignore

**Review by RishabhPatil — CHANGES_REQUESTED** [review]

File: pyclient/mkpypigenconfig.sh (hunk 1 above)
Comment (jsilberm) [reply pyclient/mkpypigenconfig.sh:41]: Done.  Thanks.

File: pyclient/mkpypigenconfig.sh (hunk 2 above)
Comment (jsilberm) [reply pyclient/mkpypigenconfig.sh:28]: Done.  Thanks.

**Comment by jsilberm** [comment]
Resolved all comments.   @RishabhPatil , please have a look.   Thanks.

**Review by RishabhPatil — APPROVED** [review]
LGTM!

**Pull request merged**